sys/Makefile.dep

```diff
@@ -0,0 +1,6 @@
+ifneq (,$(filter vaina,$(USEMODULE)))
+  USEMODULE += sock_udp
+  USEMODULE += turpial_firmware_net
+  USEMODULE += turpial_firmware_storage
+  USEMODULE += esp_idf_nvs_flash
+endif
```

Comment:
Same as in the `Makefile` file:

```
ifneq (,$(filter nvs))
  USEMODULE += turpial_firmware_storage
  USEMODULE += esp_idf_nvs_flash
endif
```

This way `storage` doesn't depends on `vaina` being included (e.g.: two separate items).